init.rb

```diff
@@ -1,4 +1,4 @@
-require 'patches/add_safety_attributes_patch'
+require File.expand_path('../lib/patches/add_safety_attributes_patch.rb', __FILE__)
```

Comment:
@ANKLV since you have mentioned that this approach isn't work correct when you try to install plugin in production mode I did a small research and found a few more variations:

```
require_dependency 'patches/add_safety_attributes_patch'
require_relative './lib/patches/add_safety_attributes_patch.rb'
```

In the patch file, try adding `require_dependency 'role'` at the top.

And the most importantly, but least logical at first glance, the placement of the require statement may affect the installation process. So try adding require 'patches/add_safety_attributes_patch' in the `init.rb` file after the `Redmine::Plugin.register` block."

Please let me know if you need additional help with this.